docs/odb_user/src/asciidoc/_chapters/load.adoc

```diff
@@ -1080,24 +1080,54 @@ define the size of this buffer in two different ways: +
 | `ifempty` | Loads the target table only if empty.
 | `nomark` | Don’t print the number of records loaded so far during loads.
 | `soe` | Stop On Error. odb stops as soon as it encounters an error.
-| `parallel=num` | odb uses as many threads as the parallel argument to extract data from partitioned source tables
-*PLUS* an equivalent number of threads to write to the target table. +
- +
- *Example* +
-  +
-If you specify `parallel=4` and the source table is made of 32 partitions, then odb start *four* threads
-(four ODBC connections) to read from the source table *PLUS* four threads (four ODBC connections) to write to the target table: +
- +
-- thread 0 extracts partitions 0-7 from source +
-- thread 1 writes data extracted from thread 0 to target +
-- thread 2 extracts partitions 8-15 from source +
-- thread 3 writes data extracted from thread 2 to target +
-- thread 4 extracts partitions 16-23 from source +
-- thread 5 writes data extracted from thread 4 to target +
-- thread 6 extracts partitions 24-31 from source +
-- thread 7 writes data extracted from thread 6 to target +
- +
-*You have to specify `splitby`.*
+| `parallel=num` | odb uses two kinds of threads: +
+
+&#8226; To extract data from partitioned source table. +
+The number of the threads is as many as the parallel argument. +
+
+&#8226; To write data to the target table. +
+The number of the threads is equal to *parallel argument* * *number of loaders*. +                                                                                                        
+
+*Example* 
+
+If you specify parallel argument = 4 and the source table is made of 32 partitions, then odb start: +
```

Comment:
Grammar: Consider "If you specify parallel argument = 4 and the source table has 32 partitions, then odb starts: +"

Reply:
Thanks Dave.
I've corrected. :)